### PR TITLE
Create epiap_test_Crossing_Equipment.xml

### PR DESCRIPTION
**Pull request by ue71603**
redo of #548

### CODE REVIEW

**Review by Aurige — APPROVED** [review]

File: examples/standards/epiap/epiap_test_Crossing_Equipment.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+						</Codespace>
+					</codespaces>
+					<dataSources>
+						<DataSource id="sbbinfoplus:DataSource_E01" version="any">
+							<Name>SBB</Name>
+						</DataSource>
+					</dataSources>
+					<equipments>
+						<CrossingEquipment version="any" id="ch:1:CrossingEquipment:850771:2">
+							<Description/>
```
Comment (skinkie) [inline examples/standards/epiap/epiap_test_Crossing_Equipment.xml:35]: Empty element.

File: examples/standards/epiap/epiap_test_Crossing_Equipment.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+				<DefaultCodespaceRef ref="sbbinfoplus"/>
+				<DefaultDataSourceRef ref="sbbinfoplus:DataSource_E01" version="any"/>
+				<DefaultLocale>
+					<TimeZoneOffset>1</TimeZoneOffset>
+					<SummerTimeZoneOffset>2</SummerTimeZoneOffset>
+					<DefaultLanguage>de</DefaultLanguage>
+				</DefaultLocale>
+			</FrameDefaults>
+			<frames>
+				<ResourceFrame id="ch:1:ResourceFrame:j21" version="any">
```
Comment (skinkie) [inline examples/standards/epiap/epiap_test_Crossing_Equipment.xml:22]: In general I dislike what you did with `version="any"` on elements (and thus references). My suggestion would be. At least make it something like "1".

Comment (ue71603) [reply examples/standards/epiap/epiap_test_Crossing_Equipment.xml:22]: adapted

File: examples/standards/epiap/epiap_test_Crossing_Equipment.xml (hunk 1 above)
Comment (ue71603) [reply examples/standards/epiap/epiap_test_Crossing_Equipment.xml:35]: done

**Comment by ue71603** [comment]
@skinkie ready for review again.

**Review by skinkie — APPROVED** [review]

**Pull request merged**